en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md

```diff
@@ -59,17 +61,19 @@ Choreo Connect's most recent version (v1.0.0) does not support all of the API Ma
 - Other
     - Immutable API artifact deploy support
     - Custom filter support at Enforcer
-    - Deploy/undeploy APIs using the command line tool APICTL
+    - Deploy/undeploy APIs using the command line tool (APICTL)
+
+## Unsupported Features
 
 The Choreo Connect currently does not support the following major functionalities. However, they will be implemented in future iterations.
 
-- Streaming APIs like SSE, web sub and etc
-- Create streaming APIs from Async API specifications
-- Analytics for the web socket APIs
+- Streaming APIs, namely SSE and WebSub/WebHook
+- Creating streaming APIs from AsyncAPI specifications
+- Analytics for the WebSocket APIs
 - GraphQL APIs
 - gRPC APIs
```

Comment:
Does API-M 4.1.0 or 4.0.0 support "gRPC APIs"?

Reply:
No, API_M does not support gRPC APIs yet.